ggml/src/ggml-cann.cpp

```diff
@@ -1220,6 +1220,73 @@ ggml_backend_cann_buffer_type(int32_t device) {
     return &ggml_backend_cann_buffer_types[device];
 }
 
+// host buffer type
```

Comment:
Add comment in Doxygen format, for all new added functions.

Reply:
OK, got it.